src/main/java/Duke.java

```diff
@@ -1,10 +1,71 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.*;
+
+public class Duke{
+    Scanner sc = new Scanner(System.in);
+    Task[] t = new Task[100];
```

Comment:
You may want to have a more definite name here for the Task array instead of just t as it might get confusing later on